src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -370,6 +370,39 @@ module Types = struct
               Stringable.Ledger_hash.to_base58_check
               @@ Staged_ledger_hash.ledger_hash staged_ledger_hash ) ] )
 
+  let natural_number_desc field_name num_bits =
```

Comment:
can you call this `uint32` and put it next to the `uint64` at the top of the file and provide a similar explanation as the uint64?

Reply:
and then all these can use this in the `typ` field